paste/cascade.py

```diff
@@ -104,7 +106,11 @@ def repl_start_response(status, headers, exc_info=None):
                         copy_len -= len(chunk)
                 f.seek(0)
             else:
-                f = StringIO(environ['wsgi.input'].read(length))
+                if six.PY3:
+                    f = BytesIO(environ['wsgi.input'].read(length))
```

Comment:
Since [six.BytesIO](https://six.readthedocs.io/index.html#six.BytesIO) is StringIO behind the scenes in py2, you ought to be able to just use `BytesIO` here, avoid the conditional, and change the three six imports to just `from six import BytesIO` as this chunk of code is the only place in this file where *IO is used.

Reply:
I was not sure this would work, so thank you for the code review!  This is a much more maintainable solution.

Reply:
> I was not sure this would work, so thank you for the code review! This is a much more maintainable solution.

Yeah, I agree this is the best outcome. I'm sorry I didn't notice this on the previous pull request. I was probably rushing :)